stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -54,23 +57,21 @@ import io.getstream.chat.android.compose.ui.util.isSingleEmoji
  *
  * @param message Message to show.
  * @param modifier Modifier for styling.
- * @param isQuote Is the message is a quote inside another message.
  * @param onLongItemClick Handler used for long pressing on the message text.
  */
 @Composable
 public fun MessageText(
     message: Message,
     modifier: Modifier = Modifier,
-    isQuote: Boolean = false,
```

Comment:
This is also a breaking change - we can't just remove or add params as we'd like. What I'd propose is duplicating this function without the `isQuote` parameter, deprecating this function, and warning users to migrate to the version which doesn't have this parameter (if they're using this)